tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -54,4 +74,12 @@ public static void processTables(
           numNodesProcessed);
     }
   }
+
+  private static void maybePrintStatus(
+      long count, String thing, String fileName, Debug.Log.Builder logCtx) {
+    if (count > 0 && count % 100000 == 0) {
+      logger.info(
+          "Processed {} {} of {}.  {}.", count, thing, fileName, LogWrapper.logSummary(logCtx));
```

Comment:
nit: maybe use count/100000 instead (or the accurate equivalent)? i was confused by this in the eia import, and didn't realize those were cumulative counts.

alternatively we can use some ansi magic to reprint over the previous line: https://stackoverflow.com/questions/45068783/java-how-to-print-output-to-previous-line-after-input/45069499

Reply:
yeah, ansi magic might not work with a logger and clobber additional metadata that gets printed out. would be cool on just stdout.

periodic dumping sounds good. i still think that having non-cumulative counts (i.e. showing instead how many lines were processed in the last n seconds) would be less confusing. but that's easily updated.

Reply:
Ah, missed that, changed to non-cumulative.